src/schemes/schemes.jl

```diff
@@ -1,3 +1,7 @@
+# Include open boundary system first,
+# since this system should not depend on specific systems.
+include("boundary/open_boundary/system.jl")
```

Comment:
That's not a reason to force some include order. Do any of the other systems depend on the open boundary system? If not, this should be irrelevant.

Reply:
Can't you just include `boundary/boundary.jl` before `fluid/fluid.jl`?

Reply:
```julia
include("boundary/open_boundary/system.jl")
include("fluid/fluid.jl")
include("boundary/boundary.jl")
```
is possible. 
But `include("boundary/boundary.jl")` before fluid is same problem  - viscosity dependencies